frontend/webapp/src/pages/index.js

```diff
@@ -17,15 +17,24 @@ export default function Home() {
     <div
       className={`${geistSans.variable} ${geistMono.variable} grid grid-rows-[20px_1fr_20px] items-center justify-items-center min-h-screen p-8 pb-20 gap-16 sm:p-20 font-[family-name:var(--font-geist-sans)]`}
     >
-      <div>
-        
+      {/* Header */}
+      <div className="text-lg font-bold">
+        <h1>Screen Recorder App</h1>
       </div>
-      <main className="flex flex-col gap-8 row-start-2">
-        + Record Video
+
+      {/* Main Content */}
+      <main className="flex flex-col gap-8 row-start-2 items-center">
+        <button
+          className="px-6 py-3 bg-blue-600 text-white rounded-lg hover:bg-blue-700 transition-all"
+          onClick={() => {
+            // You can navigate to /screen-recorder or trigger functionality here
+            window.location.href = "/screen-recorder";
```

Comment:
Please use the 'react' or 'nextjs' wat to navigate to a new page,

Reply:
@Swapnilden  Can you please address this review comment ?